lib/cc/engine/analyzers/reporter.rb

```diff
@@ -18,14 +18,49 @@ def initialize(engine_config, language_strategy, io)
         end
 
         def run
-          debug("Processing #{language_strategy.files.count} files concurrency=#{engine_config.concurrency}")
+          debug("Processing #{language_strategy.files.count} #{lang} files concurrency=#{engine_config.concurrency}")
 
           process_files
+
+          return dump_ast if engine_config.dump_ast?
```

Comment:
Hah! I picked up that style from rubocop. Sure. I can change it to something more conventional.